cmake/FindPoly.cmake

```
@@ -19,8 +19,13 @@
 include(deps-helper)
 
 find_path(Poly_INCLUDE_DIR NAMES poly/poly.h)
-find_library(Poly_LIBRARIES NAMES poly)
-find_library(PolyXX_LIBRARIES NAMES polyxx)
+if(BUILD_SHARED_LIBS)
+  find_library(Poly_LIBRARIES NAMES poly)
+  find_library(PolyXX_LIBRARIES NAMES polyxx)
+else()
+  find_library(Poly_LIBRARIES NAMES libpicpoly.a)
+  find_library(PolyXX_LIBRARIES NAMES libpicpolyxx.a)
```

```suggestion
  find_library(Poly_LIBRARIES NAMES picpoly)
  find_library(PolyXX_LIBRARIES NAMES picpolyxx)
```

Can you try if this work too? If yes, this would be the preferred way, since the suffix is set in the top-level CMakeLists.txt depending on the OS.